pkg/operator/configobserver/featuregates/observe_featuregates.go

```diff
@@ -127,7 +127,9 @@ func (f *featureFlags) getWhitelistedFeatureNames(fg *configv1.FeatureGate) ([]s
 	return newConfigValue, nil
 }
 
-func getFeaturesFromTheSpec(fg *configv1.FeatureGate) ([]string, []string, error) {
+// GetEnabledAndDisabledFeatures returns a list of enabled and disabled features for the given `FeatureGate` instance.
+// Returns an error if the object references an unknown feature set.
+func GetEnabledAndDisabledFeatures(fg *configv1.FeatureGate) ([]string, []string, error) {
```

Comment:
I don't think configobservers are a good place to host any exported funcs beyond the observer.

Reply:
@sttts would a top level `pkg/featuregates` package be a better home for these functions?